services/account_type/master_payer.yaml

```diff
@@ -274,58 +276,97 @@ Resources:
                   - 'arn:aws:s3:::${BucketName}-${Id}/*'
                   - { BucketName: !FindInMap [ Defaults, Cur, BucketName ],
                       Id: !Select [ 2, !Split [ '/', !Ref 'AWS::StackId' ] ] }
-          - Sid: CZCostMonitoring20191111
+          - Sid: CZProtect20190912
```

Comment:
NOTE: @muellerk22 and @silvexis, also I synced the JSON policies above into our automated CloudFormation policies here b/c the JSON policies are newer. Please review theses changes.